client/fingerprint/host.go

```diff
@@ -38,11 +35,7 @@ func (f *HostFingerprint) Fingerprint(cfg *config.Config, node *structs.Node) (b
 	node.Attributes["kernel.version"] = ""
 
 	if runtime.GOOS != "windows" {
```

Comment:
OK, i left it there because I assumed it was not an interesting value on Windows, but happy enable it.